### PR TITLE
Fix notifier to work with various response types.

### PR DESCRIPTION
**Pull request by vmihailenco**
Fixes https://github.com/airbrake/phpbrake/pull/21

### CODE REVIEW

**Comment by zefer** [comment]
LGTM, but getting @kyrylo eyes is probably worthwhile.

File: tests/Troublemaker.php [hunk 1]
```diff
@@ -13,4 +13,14 @@ public static function echoUndefinedVar()
...
     }
+
+    private static function doNewException()
+    {
+        return new \Exception('hello');
+    }
+
+    public static function newException()
+    {
+        return self::doNewException();
```
Comment (kyrylo) [inline tests/Troublemaker.php:24]: Did you add this to have 2 frames with predictable line numbers instead of 1?

Comment (vmihailenco) [inline tests/Troublemaker.php:24]: Exactly.

**Comment by kyrylo** [comment]
LGTM

**Pull request merged**